### PR TITLE
Updating project names to lowercase (SOC-9552)

### PR DESCRIPTION
**Pull request by asettle**
This change reflects the upstream convention for
lowercase project names. The next PR will reflect
uppercase service names.

### CODE REVIEW

**Comment by asettle** [comment]
Reviews welcome - although I am keenly aware that doing this automatically means that there are some instances I will have to fix manually (some titles, names, etc). I'll review again manually :)

File: xml/depl_require.xml [hunk 1]
```diff
@@ -1001,8 +1001,8 @@
        network storage solutions like NetApp or EMC. It is also possible to
        use local disks for block storage. A list of drivers available for
        &o_blockstore; and the features supported for each driver is
-       available from the <citetitle>CinderSupportMatrix</citetitle> at
-       <link xlink:href="https://wiki.openstack.org/wiki/CinderSupportMatrix"/>.
+       available from the <citetitle>cinderSupportMatrix</citetitle> at
+       <link xlink:href="https://wiki.openstack.org/wiki/cinderSupportMatrix"/>.
```
Comment (dmpop) [inline xml/depl_require.xml]: The page is CinderSupportMatrix

File: xml/installation-operations-upload_image.xml [hunk 2]
```diff
@@ -66,9 +66,9 @@ ansible-playbook -i hosts/verb_hosts glance-cloud-configure.yml -e proxy=&lt;PRO
   </para>
  </section>
  <section>
-  <title>Using the GlanceClient CLI to Create Images</title>
+  <title>Using the python-glanceclient CLI to Create Images</title>
   <para>
-   You can use the GlanceClient on a machine accessible to your cloud or on
+   You can use the glanceClient on a machine accessible to your cloud or on
```
Comment (dmpop) [inline xml/installation-operations-upload_image.xml]: Should it be `glanceclient`?

File: xml/planning-planning-objectstorage-swift_device_groups.xml [hunk 3]
```diff
@@ -4,9 +4,9 @@
 ]>
 <section xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude"
   xmlns:xlink="http://www.w3.org/1999/xlink" xml:id="topic-d1s-hht-tt" version="5.1">
- <title>Swift Requirements for Device Group Drives</title>
+ <title>swift Requirements for Device Group Drives</title>
```
Comment (dmpop) [inline xml/planning-planning-objectstorage-swift_device_groups.xml]:
```suggestion
 <title>Swift Requirements for Device Group Drives</title>
```

File: xml/planning-planning-objectstorage-swift_input_model.xml [hunk 4]
```diff
@@ -4,12 +4,12 @@
 ]>
 <chapter xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude"
   xmlns:xlink="http://www.w3.org/1999/xlink" xml:id="modify-input-model" version="5.1">
- <title>Modifying Example Configurations for Object Storage using Swift</title>
+ <title>Modifying Example Configurations for Object Storage using swift</title>
```
Comment (dmpop) [inline xml/planning-planning-objectstorage-swift_input_model.xml]:
```suggestion
 <title>Modifying Example Configurations for Object Storage using Swift</title>
```

File: xml/planning-planning-objectstorage-swift_zones.xml [hunk 5]
```diff
@@ -4,41 +4,41 @@
 ]>
 <section xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude"
   xmlns:xlink="http://www.w3.org/1999/xlink" xml:id="designing-swift-zones" version="5.1">
- <title>Designing Swift Zones</title>
+ <title>Designing swift Zones</title>
```
Comment (dmpop) [inline xml/planning-planning-objectstorage-swift_zones.xml]:
```suggestion
 <title>Designing Swift Zones</title>
```

File: xml/planning-planning-objectstorage-swift_zones.xml [hunk 6]
```diff
@@ -218,9 +218,9 @@
       - AZ4</screen>
  </section>
  <section xml:id="zones-ring-level">
-  <title>Specifying Swift Zones at Ring Level</title>
+  <title>Specifying swift Zones at Ring Level</title>
```
Comment (dmpop) [inline xml/planning-planning-objectstorage-swift_zones.xml]:
```suggestion
  <title>Specifying Swift Zones at Ring Level</title>
```

File: xml/planning-planning-rec_min_ironic.xml [hunk 7]
```diff
@@ -4,7 +4,7 @@
 ]>
 <section xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude"
   xmlns:xlink="http://www.w3.org/1999/xlink" xml:id="rec-min-ironic" version="5.1">
- <title>Recommended Hardware Minimums for an Ironic Flat Network Model</title>
+ <title>Recommended Hardware Minimums for an ironic Flat Network Model</title>
```
Comment (dmpop) [inline xml/planning-planning-rec_min_ironic.xml]:
```suggestion
 <title>Recommended Hardware Minimums for an Ironic Flat Network Model</title>
```

File: xml/planning-planning-rec_min_swift.xml [hunk 8]
```diff
@@ -4,7 +4,7 @@
 ]>
 <section xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude"
   xmlns:xlink="http://www.w3.org/1999/xlink" xml:id="rec-min-swift" version="5.1">
- <title>Recommended Hardware Minimums for an Entry-scale Swift Model</title>
+ <title>Recommended Hardware Minimums for an Entry-scale swift Model</title>
```
Comment (dmpop) [inline xml/planning-planning-rec_min_swift.xml]:
```suggestion
 <title>Recommended Hardware Minimums for an Entry-scale Swift Model</title>
```

File: xml/security-barbican_admin.xml [hunk 9]
```diff
@@ -289,7 +289,7 @@ ansible-playbook -i hosts/verb_hosts barbican-reconfigure-credentials-change.yml
   </procedure>
  </section>
  <section>
-  <title>Checking Barbican Status</title>
+  <title>Checking barbican Status</title>
```
Comment (dmpop) [inline xml/security-barbican_admin.xml]:
```suggestion
  <title>Checking Barbican Status</title>
```

File: xml/security-dar.xml [hunk 10]
```diff
@@ -186,15 +186,15 @@ ansible-playbook -i hosts/verb_hosts barbican-reconfigure.yml</screen>
   </orderedlist>
  </section>
  <section>
-  <title>Configuring Cinder volumes for encryption</title>
+  <title>Configuring cinder volumes for encryption</title>
```
Comment (dmpop) [inline xml/security-dar.xml]:
```suggestion
  <title>Configuring Cinder Volumes for Encryption</title>
```

File: xml/security-horizon_ssl_auth.xml [hunk 11]
```diff
@@ -25,10 +25,10 @@
   </para>
  </note>
  <section>
-  <title>Keystone configuration</title>
+  <title>keystone configuration</title>
```
Comment (dmpop) [inline xml/security-horizon_ssl_auth.xml]:
```suggestion
  <title>Keystone Configuration</title>
```

File: xml/security-horizon_ssl_auth.xml [hunk 12]
```diff
@@ -361,9 +361,9 @@ ansible-playbook -i hosts/verb_hosts FND-CLU-reconfigure.yml</screen>
   </note>
  </section>
  <section xml:id="section-qjc-42p-hx">
-  <title>Horizon configuration</title>
+  <title>horizon configuration</title>
```
Comment (dmpop) [inline xml/security-horizon_ssl_auth.xml]:
```suggestion
  <title>Horizon Configuration</title>
```

File: xml/soc-monitoring-operations-components.xml [hunk 13]
```diff
@@ -45,30 +45,30 @@
...
-  notification engine of Monasca. For details, refer to the
-  <link xlink:href="https://wiki.openstack.org/wiki/Monasca">Monasca Wiki</link>.
+  notification engine of monasca. For details, refer to the
+  <link xlink:href="https://wiki.openstack.org/wiki/monasca">monasca Wiki</link>.
  </para>
 </section>
 
  <section xml:id="horizon-plugin">
-  <title>Horizon Plugin</title>
+  <title>horizon Plugin</title>
```
Comment (dmpop) [inline xml/soc-monitoring-operations-components.xml]:
```suggestion
  <title>Horizon Plugin</title>
```

File: xml/soc-operations-glossary.xml [hunk 14]
```diff
@@ -77,7 +77,7 @@
   description of the data to be monitored. Metrics values represent the actual
   monitoring data that is collected and presented in &productmonitoring;.
  </para>
- <bridgehead renderas="sect4">Monasca</bridgehead>
+ <bridgehead renderas="sect4">monasca</bridgehead>
```
Comment (dmpop) [inline xml/soc-operations-glossary.xml]:
```suggestion
 <bridgehead renderas="sect4">Monasca</bridgehead>
```

File: xml/installation-installation-installation_troubleshooting.xml [hunk 15]
```diff
@@ -224,14 +224,14 @@ not larger than existing size (7629 extents)</screen>
    files is based on a 500 GB disk, so if your logical volumes are smaller you
    may need to increase the percentage so there is enough room.
   </para>
-  <bridgehead renderas="sect4">Multiple Keystone Failures Received during site.yml</bridgehead>
+  <bridgehead renderas="sect4">Multiple keystone Failures Received during site.yml</bridgehead>
```
Comment (dmpop) [inline xml/installation-installation-installation_troubleshooting.xml]:
```suggestion
  <bridgehead renderas="sect4">Multiple Keystone Failures Received during site.yml</bridgehead>
```

File: xml/installation-installation-ironic-install_ironic_overview.xml [hunk 16]
```diff
@@ -6,9 +6,9 @@
  xmlns="http://docbook.org/ns/docbook" version="5.1"
  xmlns:xi="http://www.w3.org/2001/XInclude"
  xmlns:xlink="http://www.w3.org/1999/xlink">
- <title>Installing Baremetal (Ironic)</title>
+ <title>Installing Baremetal (ironic)</title>
```
Comment (dmpop) [inline xml/installation-installation-ironic-install_ironic_overview.xml]:
```suggestion
 <title>Installing Baremetal (Ironic)</title>
```

File: xml/metering-glance_meters.xml [hunk 17]
```diff
@@ -6,7 +6,7 @@
  <!ENTITY % entities SYSTEM "entity-decl.ent"> %entities;
 ]>
 <section xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xlink="http://www.w3.org/1999/xlink" xml:id="glance-meters">
- <title>Image (Glance) Meters</title>
+ <title>Image (glance) Meters</title>
```
Comment (dmpop) [inline xml/metering-glance_meters.xml]:
```suggestion
 <title>Image (Glance) Meters</title>
```

File: xml/operations-central_log_configure_services.xml [hunk 18]
```diff
@@ -780,7 +780,7 @@ keystone_logstash_loglevel: INFO</screen>
   </procedure>
  </section>
  <section xml:id="Loglvl-ironic">
-  <title>Ironic</title>
+  <title>Bare Metal(ironic)</title>
```
Comment (dmpop) [inline xml/operations-central_log_configure_services.xml]:
```suggestion
  <title>Bare Metal (ironic)</title>
```

File: xml/operations-configuring-identity-keystone_federation.xml [hunk 19]
```diff
@@ -6,17 +6,17 @@
...
- <title>Keystone-to-Keystone Federation</title>
+ <title>keystone-to-keystone Federation</title>
  <para>
-  This topic explains how you can use one instance of Keystone as an identity
+  This topic explains how you can use one instance of keystone as an identity
   provider and one as a service provider.
  </para>
  <section>
-  <title>What Is Keystone-to-Keystone Federation?</title>
+  <title>What Is keystone-to-keystone Federation?</title>
```
Comment (dmpop) [inline xml/operations-configuring-identity-keystone_federation.xml]:
```suggestion
  <title>What Is Keystone-to-Keystone Federation?</title>
```

**Comment by asettle** [comment]
> good illustrations of how capitalization should work. And reminder to watch for ongoing.

Thanks! This should do everything. I'll go ahead and merge now. The next step will be to edit the service names.

**Pull request merged**